{"name": "Create order with invalid ingredient hash", "status": "failed", "statusDetails": {"message": "AssertionError: assert 403 == 500\n +  where 403 = <Response [403]>.status_code", "trace": "self = <Diplom_2.tests.test_create_order.TestCreateOrder object at 0x00000128F94960F0>\n\n    @allure.title('Create order with invalid ingredient hash')\n    def test_create_order_invalid_ingredients(self):\n        response = self.api.create_order(['invalid_hash'], self.token)\n>       assert response.status_code == 500\nE       assert 403 == 500\nE        +  where 403 = <Response [403]>.status_code\n\ntests\\test_create_order.py:48: AssertionError"}, "start": 1751986478004, "stop": 1751986478366, "uuid": "a6b186f6-54be-4dcc-94b9-140242cc296c", "historyId": "35c4c4f479bb1208f0fc34c58f8f1e92", "testCaseId": "35c4c4f479bb1208f0fc34c58f8f1e92", "fullName": "tests.test_create_order.TestCreateOrder#test_create_order_invalid_ingredients", "labels": [{"name": "feature", "value": "Create Order"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_order"}, {"name": "subSuite", "value": "TestCreateOrder"}, {"name": "host", "value": "DESKTOP-42BDC19"}, {"name": "thread", "value": "24480-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_order"}]}